recipe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "imbalanced-ensemble" %}
+{% set version = "0.2.1" %}
```

Comment:
It's suspicious that v0.2.1 doesn't have the `requirements.txt` file, see https://inspector.pypi.io/project/imbalanced-ensemble/0.2.1/packages/6d/f9/38f13157b20a77acfa08cd2a67148b4d23d10d0768a4b448e1df24bc601d/imbalanced-ensemble-0.2.1.tar.gz/. The same applies for 0.2.2.
The upstream source code does not provide tags for >=0.2.0. Hmmm...

Could you build the latest version **0.2.2**? But then you need to modify requirements.txt, see https://inspector.pypi.io/project/imbalanced-ensemble/0.2.2/packages/33/29/d5be3c99394d8a941305d6245bafabce8ca5a01caae228a988258828e4f1/imbalanced-ensemble-0.2.2.tar.gz/imbalanced-ensemble-0.2.2/imbalanced_ensemble.egg-info/requires.txt

Reply:
the latest version requires `scikit-learn>=1.5.0`, which we do not provide yet. highest version we have is `1.4.2`